src/Guide/Api/Server.hs

```diff
@@ -76,19 +79,21 @@ fullServer db =
 -- | Serve the API on port 4400.
 --
 -- You can test this API by doing @withDB mempty runApiServer@.
-runApiServer :: AcidState GlobalState -> IO ()
-runApiServer db = do
-  say "API is running on port 4400"
-  run 4400 $ corsPolicy $ serve (Proxy @FullApi) (fullServer db)
+runApiServer :: Config -> AcidState GlobalState -> IO ()
+runApiServer Config{..} db = do
+  say $ T.concat ["API is running on port ", toText $ show _portApi]
+  run _portApi $ corsPolicy Config{..} $ serve (Proxy @FullApi) (fullServer db)
   where
-    corsPolicy :: Middleware
-    corsPolicy = cors (const $ Just policy)
-    policy :: CorsResourcePolicy
-    policy = simpleCorsResourcePolicy
-                -- TODO: Add Guide's frontend address (and maybe others resources)
-                -- to list of `corsOrigins` to allow CORS requests
-                { corsOrigins = Just ([
-                      "http://localhost:3333" -- Guide's frontend running on localhost
-                    , "http://localhost:4400" -- The /api endpoint
-                    ], True)
-                }
+    corsPolicy :: Config -> Middleware
```

Comment:
`corsPolicy` already has access to the `Config`, you don't have to pass it here. All function parameters are accessible in `where`.